packages/react/toast/src/Toast.stories.tsx

```diff
@@ -478,7 +478,7 @@ const viewportClass = css({
 });
 
 const rootClass = css({
-  all: 'unset',
+  listStyle: 'none',
```

Comment:
@benoitgrelard this wasn't what was making the styles super bad (a `yarn reset` mostly fixed that) but i realised the `all: unset` was breaking some other styles i missed so just resetting the necessary bits only.

Reply:
Yeah it's an issue with this version of stitches we have in, that's why I tried to upgrade it a while ago to avoid that.